osf/models/metadata.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import jsonschema
+from django.db import models
+
+from framework.auth.core import Auth
+from framework.exceptions import PermissionsError
+from addons.osfstorage.models import OsfStorageFile
+from api.base.schemas.utils import from_json
+from osf.models import NodeLog
+from osf.models.base import BaseModel, ObjectIDMixin
+from osf.models.metaschema import FileMetadataSchema
+from osf.utils import permissions as osf_permissions
+from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
+from osf.metadata.serializers import serializer_registry
+from website.util import api_v2_url
+
+
+def validate_user_entered_metadata(value):
```

Comment:
How about just `validate_metadata`?

Reply:
True, since everything in metadata is user entered!